### DISCUSSION TITLE
Test flight Beta is full

### DISCUSSION THREAD

**Discussion by daycrailor** [creepymonster/GlucoseDirect#536] [General]
The app looks great, but currently the beta is full. Is there any chance to join the beta too? 
Thanks!!

**Comment by hestovall** [comment]
Same question here.

**Comment by devxor** [comment]
Same here.

**Comment by Baodor** [comment]
Here too

**Comment by KoRnAlive** [comment]
I have the same problem :(

**Comment by jobaxp** [comment]
Yes, the same here will there be more openings beta testers anytime soon, how will we be notified?
Thanks!

**Comment by digidoc2** [comment]
Hi Reimar,

is it possible to invite me, please?

Thanka
Rob

**Reply by Baodor** [reply]
I am waiting to, not even for me but no answer

**Comment by daycrailor** [comment]
Im in. Its awesome.

**Comment by jobaxp** [comment]
How did you get in?Op 5 aug. 2023 om 09:01 heeft daycrailor ***@***.***> het volgende geschreven:﻿
Im in. Its awesome.

**Reply by srini4000** [reply]
Is it possible to invite me as well?

**Comment by jobaxp** [comment]
Join the Glucose Direct betatestflight.apple.comOp 23 aug. 2023 om 18:45 heeft srini4000 ***@***.***> het volgende geschreven:﻿
Is it possible to invite me as well?

**Comment by 3djc** [comment]
Would love to join too. Libre 2 user, Libre 3 in 2 sensors, with iPhoneXR / Watch 8 / iPad Pro. Can help with french translations

For invite: pleaseinviteme@legidata.com

**Comment by redexit1** [comment]
Could I join in, please?

**Comment by scottm7** [comment]
I would also like to get an invite to the beta if possible.  Thanks!

**Comment by lmotzo** [comment]
Hi Reimar,

i am a software developer and T1. A little bit disappointed of about waiting at Abbott is it possible to invite me to the Glucosedirect project?

Thanks

**Comment by kesslerj** [comment]
Hi @creepymonster,

is there any way to be part of the Beta phase at the moment?

Thanks

**Comment by lmotzo** [comment]
Hi Jonas, i world love to join and support the Test fight Beta. Unfortunately I was not  invited myself sogar - good luck.  Rgds Luca Von meinem Smartphone gesendetAm 12.12.2023 um 15:06 schrieb Jonas Keßler ***@***.***>:﻿
Hi @creepymonster,
is there any way to be part of the Beta phase at the moment?
Thanks

**Comment by chrismon** [comment]
I would love an invite as well.

**Comment by bobdowns** [comment]
Would also like to join the beta, please.

**Comment by Boky3000** [comment]
Aside of a full beta and would love to join…is there any planned moment to release the app to the normal App Store?

**Comment by gfrino** [comment]
Same here, would like to join the beta testing in testflight: gfrino@gmail.com

**Comment by nileshsamin** [comment]
Is it possible for me to get an invite to the beta test? Thanks
